android/cli/commands/_build.js

```diff
@@ -3206,7 +3206,8 @@ AndroidBuilder.prototype.generateI18N = async function generateI18N() {
 		root.appendChild(dom.createTextNode('\n'));
 
 		// Create the XML file under the Android "res/values-<locale>" folder.
-		const localeSuffixName = (locale === 'en' ? '' : '-' + resolveRegionName(locale));
+		const defaultLang = this.tiapp.defaultLang ? this.tiapp.defaultLang : 'en';
```

Comment:
How about this?

```suggestion
		const defaultLang = this.tiapp.defaultLang || 'en';
```

Reply:
tested and works too  and looks better :)